docs/infra/set-up-aws-account.md

```diff
@@ -37,6 +37,14 @@ make infra-set-up-account ACCOUNT_NAME=<ACCOUNT_NAME>
 
 This command will create the S3 tfstate bucket and the GitHub OIDC provider. It will also create a `[account name].[account id].s3.tfbackend` file in the `infra/accounts` directory.
 
+### 3. Check that GitHub actions can authenticate into the AWS account
+
+This step requires [GitHub CLI](https://cli.github.com/) to be installed. If you don't have it, you can install on Mac via `brew install gh`
+
+```bash
+make infra-check-github-actions-auth ACCOUNT_NAME=<ACCOUNT_NAME>
```

Comment:
```suggestion
# ACCOUNT_NAME is the name of account in infra/accounts
make infra-check-github-actions-auth ACCOUNT_NAME=<ACCOUNT_NAME>
```

I missed this note on my first skim, so I imagine other people will too

Reply:
did you miss it when skimming the PR changes or when skimming the whole doc? the ACCOUNT_NAME is defined in Step 2 already so it seems redundant to re-explain it in Step 3.